cylc/flow/task_remote_mgr.py

```diff
@@ -153,28 +157,31 @@ def subshell_eval_reset(self):
             if value is not None:
                 del self.remote_command_map[key]
 
-    def remote_init(self, platform, curve_auth, client_pub_key_dir):
+    def remote_init(
+            self, platform: Dict[str, Any], curve_auth: 'ThreadAuthenticator',
+            client_pub_key_dir: str) -> None:
         """Initialise a remote host if necessary.
 
         Call "cylc remote-init" to install suite items to remote:
             ".service/contact": For TCP task communication
             "python/": if source exists
 
         Args:
-            curve_auth (ThreadAuthenticator):
-                The ZMQ authenticator.
-            client_pub_key_dir (str):
-                Client public key directory, used by the ZMQ authenticator.
-            platform (dict):
-                A dictionary containing settings relating to platform used in
+            platform: A dict containing settings relating to platform used in
                 this remote installation.
+            curve_auth: The ZMQ authenticator.
+            client_pub_key_dir: Client public key directory, used by the
+                ZMQ authenticator.
 
         """
         install_target = platform['install target']
-
+        if install_target == 'localhost':
+            self.remote_init_map[install_target] = REMOTE_INIT_NOT_REQUIRED
```

Comment:
`REMOTE_FILE_INSTALL_DONE` or `REMOTE_INIT_DONE`?

Reply:
Just to make a note after our team meeting: it should be `REMOTE_FILE_INSTALL_DONE` because this status implies `REMOTE_INIT_DONE`, i.e. it it indicates further completion than `REMOTE_INIT_DONE`